src/components/side_panel/chart/building_blocks/general_design/general_design_editor.ts

```diff
@@ -22,6 +22,7 @@ interface Props {
   figureId: UID;
   definition: ChartDefinition;
   updateChart: (figureId: UID, definition: Partial<ChartDefinition>) => DispatchResult;
+  canUpdateChart: (figureId: UID, definition: Partial<ChartDefinition>) => DispatchResult;
```

Comment:
could also create an interface DefaultChartProps or something, and have everything using/extending it. It's now used at like 20 places